### PR TITLE
DOC: Suggest replacement for tostring_rgb

### PR DESCRIPTION
**Pull request by larsoner**
We hit this `DeprecationWarning` in MNE-Python (EDIT: After https://github.com/matplotlib/matplotlib/commit/9c79686a3081f9928daedaeac423058a8ab1483d) and I had to go to the commits to see the comment that `argb` should cover most use cases.

If this makes sense to add then I'll modify the actual warning itself, too, which would have made finding the workaround even faster.

### CODE REVIEW

**Comment by oscargus** [comment]
I think this makes sense!

File: doc/api/next_api_changes/deprecations/XXXXX-AL.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ``RendererAgg.tostring_rgb`` and ``FigureCanvasAgg.tostring_rgb``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-... are deprecated with no replacement.
+These have no direct replacement. Consider using ``tostring_argb`` instead,
```
Comment (timhoffm) [inline doc/api/next_api_changes/deprecations/XXXXX-AL.rst:3]:
```suggestion
... are deprecated with no direct replacement. Consider using ``tostring_argb`` instead,
```

**Comment by tacaswell** [comment]
I am 👍🏻 on this and adding the note to the message itself as well.

**Comment by anntzer** [comment]
Note that it was also suggested to deprecate tostring_argb (https://github.com/matplotlib/matplotlib/pull/25484#pullrequestreview-1346176204).  In the use case of mne-python (https://github.com/mne-tools/mne-python/pull/11574/?) I would suggest using e.g. `np.array(canvas.buffer_rgba())` instead.

**Comment by larsoner** [comment]
Okay pushed an updated version!

**Comment by larsoner** [comment]
@QuLogic done!

**Review by QuLogic — APPROVED** [review]

**Pull request merged**